ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-hosted-pipeline.yml

```diff
@@ -1056,6 +1070,8 @@ spec:
         - evaluate-preflight-result
       taskRef:
         name: get-ci-results
+      when:
```

Comment:
This can't be skipped using a when because the result of this task is used in other places.